types/scripts/tools.ts

```diff
@@ -43,6 +43,7 @@ export interface Filters {
   technology?: Array<string>;
   categories: Array<Category>;
   hasCommercial?: boolean;
+  isAsyncAPIOwner?: boolean | string;
```

Comment:
Why did you define this as sa tring? This should always be boolean.

Reply:
We had an empty string as a default value of isAsyncAPIOwner parameter 
https://github.com/asyncapi/website/blob/1df5c6b91851d047a4ab6859516e87cc1bdd87cd/scripts/tools/tools-object.ts#L46

I guess, this does not serve any intention. I will fix it.